### ISSUE TITLE
Stepper section have some redundant focus with arrow key navigation

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4658]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 2.4.3 Focus Order

**Test Environment:**
- Test URL: https://clarity.design/documentation/stepper
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 2020.1

**Steps to reproduce:**
1. Open the test URL.
2. Go to stepper section and try to navigate through it using arrow key.

**Expected result:**
User should be able to navigate through the Stepper content appropriately without any redundancy or verbose announcement.

**Actual result:**

- Collapsed Stepper section have multiple focus with arrow key navigation due to which redundant announcement is made in the following way for collapsed stepper 1.
1. Button collapsed Success.
2. Button collapsed 1.
3. Button collapsed Legal Name.
4. Button collapsed Description goes here.

- Expanding the button and moving one step forward again announces the stepper section button content.

Similar issue is observed for Left navigation buttons in Wizard component.

**Proposed solution by @chlane **
Implement like an ARIA Disclosure https://www.w3.org/TR/wai-aria-practices-1.1/examples/disclosure/disclosure-faq.html
